javascript/javascript1/week1/house_price.js

```diff
@@ -0,0 +1,18 @@
+function housePrice(width,depth,height, gardenSizeInM2){
+    const volumeInMeters = width*depth*height;
+    return volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+}
```

Comment:
add spaces after the commas in the function parameter list.

Also here add spaces between the operators in line `2`

You could name `volumeInMeters` to `volumeInM3` to be consistent.